Specs/DateTime/English/DateTimeModel.json

```diff
@@ -19448,5 +19448,37 @@
         }
       }
     ]
+  },
+  {
+    "Input": "My temperature was 37.1 in the morning",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "Results": [
+      {
+        "Text": "in the morning",
+        "Start": 24,
+        "End": 37,
+        "TypeName": "datetimeV2.timerange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "TMO",
+              "type": "timerange",
+              "start": "08:00:00",
+              "end": "12:00:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Labs were notable for a BUN/Cr of 30/4.85, troponin 0.182",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
```

Comment:
Can the filter be easily added in the other platforms?

Reply:
I will check if it is possible

Reply:
added it to Java and Python, in Javascript the method validateMatch is not implemented.